architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,29 +175,143 @@ endif::[]
 [[image-streams]]
 == Image Streams
 
-An _image stream_ comprises any number of
-xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker-formatted
-container images] identified by tags. It presents a single virtual view of
-related images, similar to an image repository, and may contain images from any
-of the following:
+An _image stream_ is a mapping between tags in {product-title} objects and 
+xref:../../architecture/core_concepts/containers_and_images.adoc#docker-images[Docker image] images. 
+You can think of an image stream as a wrapper or abstraction
+for Docker images to see what images are available and to ensure that you are using the specific image you need even if the image in 
+the repository changes.
+Image streams do not contain actual image data, but presenta single virtual view of related images, similar to an image repository. 
 
-. Its own image repository in {product-title}'s
-xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
+Yoyu can  configure xref:../dev_guide/builds/triggering_builds.adoc#image-change-triggers[Builds] and
+xref:../dev_guide/deployments/basic_deployment_operations.adoc#image-change-trigger[Deployments] to watch an image stream
+for notifications when new images are added and react by performing a Builds or Deployment, respectively.
+
+For example, if a Deployment is using a certain image and a new version of that image is created, a Deployment could be automatically performed.
+
+//from https://blog.openshift.com/image-streams-faq/
+This means that even if the source image changes, the image stream will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+The source images can be stored in any of the following:
+
+* {product-title}'s xref:../../architecture/infrastructure_components/image_registry.adoc#integrated-openshift-registry[integrated
 registry]
-. Other image streams
-. Image repositories from external registries
+* An external registry, for example `registry.access.redhat.com` or `hub.docker.com`
+* Other image streams
 
-ifdef::openshift-origin,openshift-online,openshift-enterprise,openshift-dedicated[]
-Image streams can be used to automatically perform an action when new images are
-created. Builds and deployments can watch an image stream to receive
-notifications when new images are added and react by performing a build or
-deployment, respectively. For a
-curated set of image streams, see the
+The following image stream contains two tags: `34` which points to a Python v3.4 image and `35` which points to a Python v3.5:
+
+When you define an image stream object in a cluster, you point to an image stream tag, not the Docker repository. When you build or deploy
```

Comment:
what's an image stream object?

Do you mean an object that references an image stream tag?  (such as a deploymentconfig or buildconfig)?